man/udev.xml

```diff
@@ -279,6 +279,14 @@
                     for possible values.</para>
                   </listitem>
                 </varlistentry>
+                <varlistentry>
+                  <term><literal>conf-virt</literal></term>
```

Comment:
as in the other places, it would be better to spell out 'confidential' as usual 'conf' means 'configuration'

Reply:
maybe call this "cvm"?

Reply:
Yep, `cvm` would match the `--cvm` flag to `systemd-detect-virt` which the docs here point to.